src/client/src/routes/SimpleLauncher/Link.tsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react'
+import { styled, Styled } from 'rt-theme'
+
+declare const global: any
+
+export interface LinkProps {
+  is?: React.ComponentType<any>
+  to: any
+  [key: string]: any
+}
+
+const LinkRoot: Styled<{ onClick: any }> = styled.button`
```

Comment:
Type this

Reply:
It's kind of bonkers to type a standard API of React and a DOM element which ninety-nine percent of Styled Components will resolve to — right? 

It's beyond the concern of a Styled Component — if you can point out where the original typings fail I'd love to PR on that.